app_test.go

```diff
@@ -334,6 +342,28 @@ func TestBashCompletionOptions(t *testing.T) {
 			Args:            "--completion-bash two --flag-3 opt4 --",
 			ExpectedOptions: []string{"--help", "--flag-2", "--flag-3", "--flag-0", "--flag-1"},
 		},
+
+		// Args complete
+		{
+			// After a command with arg options
+			Args:            "--completion-bash three ",
+			ExpectedOptions: []string{"arg-2-opt-1", "arg-2-opt-2", "arg-3-opt-1", "arg-3-opt-2"},
```

Comment:
Good point. I'll have to spend a bit more time understanding the parsing logic to plumb down where in the process it is to the code that knows where to get completion options from. I'll get on that.